### PR TITLE
Archivist splitup

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Replaces the Archives with a Server Room.
Replaces the Archivist with 2 new roles:
- IT Technician: manages the server rooms (1 position)
- RAISA Agent: protects against espionage (1 position)

## Why It's Good For The Game

The archivist role just doesn't make sense lol, and I needed a server room anyways so I just uhhh swapped it out. The RAISA agent doesn't really have all that much to do but I hope more is added over time.

## Changelog

:cl:
add: Adds the IT Technician role, an engineering side-role responsible for keeping the server room operational and safe from malware.
add: Adds the RAISA Agent role, a high-level EZ security role that protects against corporate espionage.
add: Overhauls the archives into a server room. Currently, the only server is a database storing all SCP files.
add: Converts all SCP files into digital files.
del: Removes the Archivist role.
/:cl:

### CODE REVIEW

**Comment by TheDarkElites** [comment]
dont name it IT technician. Their job will still include the maintenance of files and entries on the server, not just fixing technical issues. They have more authority than that.

**Comment by aidan-bowman** [comment]
The RAISA agent can't access the server room, need to fix this

File: code/game/jobs/job/engineering.dm [hunk 1]
```diff
@@ -280,3 +280,45 @@
 	mechanical_difficulty = "Medium - Hard"
 	duties = "Manage the Engineering department. Delegate construction and repair work. Facilitate larger projects."
 	codex_guides = list("<l>Hacking Wires</l>")
+
+/datum/job/it_tech
+	title = "IT Technician"
+	department = "Civilian"
```
Comment (EgorDinamit) [inline code/game/jobs/job/engineering.dm]: Fix department.

**Comment by TerryTehCoder** [comment]
> The RAISA agent can't access the server room, need to fix this

The I.T Technician cannot access the server room either. Spawned in the area with the database computers and was stuck in there waiting for the AIC to open the door, no access to the windoors either.

**Comment by checkraisefold** [comment]
image
still need to fix it tech department please

File: code/game/jobs/job/engineering.dm (hunk 1 above)
Comment (aidan-bowman) [reply code/game/jobs/job/engineering.dm]: fixed

**Comment by aidan-bowman** [comment]
fixed access shit, this codebase is so fucking dumb

**Comment by aidan-bowman** [comment]
for some reason the commit isn't showing up?

**Pull request merged**